curriculum/train_helper.py

```diff
@@ -0,0 +1,52 @@
+"""this is a place holder for dummy functions"""
+# pylint: disable=all
+
+import random
+import numpy as np
+
+
+######################################################################
+# to be provided by Joseph
+#from cleanrl_ppo_lstm import train_on_tasks, evaluate_on_tasks
```

Comment:
These will just be the train and evaluate methods of clean_pufferl once I add tasks as an arg

Reply:
Waiting for it to be finalized. The end-to-end script and env have been modified to work with a curriculum file, containing the whole training tasks and sampling training tasks on reset()